City-of-Helsinki/drupal-hdbt
# UHF-1733: Add new spacing variable, change paragraph title margins to…

… bottom, fix line-height to 1 on all headers

To test:

1. Checkout the branch for your hdbt-theme folder. Any site should work fine.
2. `make drush-cr`
3. Go to check columns, list of links and content card paragraphs that if they have headers written on them the margin is on bottom of the header. Also check that all headers h1-h3 have line-height 1.
